### PR TITLE
fix(DB/creature): Ras Frostwhisper should move around

### PR DESCRIPTION
**Pull request by Zoidwaffle**
## Changes Proposed:
-  Ras Frostwhisper should move around

## Issues Addressed:
- This is cosmetic, but he moves around here: https://www.youtube.com/watch?v=SQhoL0aPwaM&start=2289
- This is taken from [TC](https://github.com/TrinityCore/TrinityCore/commit/e33384d4ddbd4477f8cc5cae6e0679ab914595a5) and all credits go to the original authors

## Tests Performed:
- Applied SQL, watched in awe as Ras Frostwhisper moves around

## How to Test the Changes:

- `.go c id 10508`
- watch Ras Frostwhisper for a while

## Known Issues and TODO List:

- [ ]
- [ ] 

## Target Branch(es):
- [x] Master

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by pklloveyou** [comment]
Tested

File: data/sql/updates/pending_db_world/rev_1612870829159186719.sql [hunk 1]
```diff
@@ -0,0 +1,17 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1612870829159186719');
+
+-- Ras Frostwhisper movement
+
+UPDATE `creature` SET `MovementType` = 2 WHERE `id` = 10508;
+
+DELETE FROM `creature_addon` WHERE `guid` IN (SELECT `guid` FROM `creature` WHERE `id` = 10508);
+UPDATE `creature_template_addon` SET `path_id` = 488500, `bytes2` = 4097 WHERE `entry` = 10508;
```
Comment (pklloveyou) [inline data/sql/updates/pending_db_world/rev_1612870829159186719.sql:8]:
```sql
UPDATE `creature_addon` SET `path_id` = 488500, `bytes2` = 4097 WHERE `entry` = 10508;
```

Comment (pklloveyou) [inline data/sql/updates/pending_db_world/rev_1612870829159186719.sql:8]: UPDATE `creature_addon` SET `path_id` = 488500, `bytes2` = 4097 WHERE `guid` = 10508;

**Comment by BarbzYHOOL** [comment]
Can you add  offl as co-author @Zoidwaffle ? and also thank you for this!!

`git commit --amend --no-edit  --author="Someone <email@email.com>"` should be enough for the PR

**Comment by Zoidwaffle** [comment]
@BarbzYHOOL - it's a good idea and it has been added now!

**Comment by FrancescoBorzi** [comment]
> Can you add offl as co-author @Zoidwaffle ? and also thank you for this!!
> 
> `git commit --amend --no-edit --author="Someone <email@email.com>"` should be enough for the PR
> 
> EDIT: or better `Co-authored-by: name <name@example.com>` I think we should add this to the git commit template

no need to commit, just add `Co-authored-by: name <name@example.com>` in the PR description

**Comment by Kitzunu** [comment]
Problem is when you do that, not everyone that merges will add it to the commit description

**Pull request merged**